yt_dlp/extractor/twitter.py

```diff
@@ -336,7 +336,7 @@ class TwitterIE(TwitterBaseIE):
             'id': '665052190608723968',
             'display_id': '665052190608723968',
             'ext': 'mp4',
-            'title': 'md5:e99588f17b3dd0503814ffb560e64731',
+            'title': str,
```

Comment:
?

Reply:
`title` is composed of `{author} - {description}` where on this account the `author` constantly changes, hence why I was putting it as `str`. Alternatively these are also sensible:
- Use the full description as string instead so the test fail diff shows that only the author name changed
- Regex match only the start , copying the `author` regex
- Just skip the test bc its unreliable.

Reply:
imo regex is the best option here. use some of the actual description and let the rest be .* so we know we are extracting the right field